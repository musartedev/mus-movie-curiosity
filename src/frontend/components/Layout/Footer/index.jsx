import React from 'react';

import '../../../styles/components/Layout/Footer.styl';

const Footer = () => {
  return (
    <footer className="Footer">
      <span>Made with</span>
      <span aria-label="love" role="img">
        ðŸ’š
      </span>
      <span>from</span>
      <a
        href="https://github.com/musartedev"
        target="_blank"
        rel="noopener noreferrer"
      >
        MariangÃ©lica Useche
      </a>
      <span>to</span>
      <a href="https://platzi.com/" target="_blank" rel="noopener noreferrer">
        Platzi
      </a>
    </footer>
  );
};

export default Footer;
