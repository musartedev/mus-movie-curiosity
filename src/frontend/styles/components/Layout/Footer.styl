@import '../../variables.styl'

.Footer
  display flex
  align-items center
  justify-content center
  background-color $background
  color $white

  a
    margin 0 0.25em