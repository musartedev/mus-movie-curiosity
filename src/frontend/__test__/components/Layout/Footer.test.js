import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../../components/Layout/Footer';

describe('<Footer />', () => {
  test('Footer component should render', () => {
    const footer = mount(<Footer />);
    expect(footer.length).toEqual(1);
  });

  test('Header snapshot should match', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
