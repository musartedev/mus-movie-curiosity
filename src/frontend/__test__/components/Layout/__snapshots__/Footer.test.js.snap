// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Header snapshot should match 1`] = `
<footer
  className="Footer"
>
  <span>
    Made with
  </span>
  <span
    aria-label="love"
    role="img"
  >
    ðŸ’š
  </span>
  <span>
    from
  </span>
  <a
    href="https://github.com/musartedev"
    rel="noopener noreferrer"
    target="_blank"
  >
    MariangÃ©lica Useche
  </a>
  <span>
    to
  </span>
  <a
    href="https://platzi.com/"
    rel="noopener noreferrer"
    target="_blank"
  >
    Platzi
  </a>
</footer>
`;
